tm2/pkg/bft/rpc/lib/server/handlers.go

```diff
@@ -14,6 +14,8 @@ import (
 	"strings"
 	"time"
 
+	"golang.org/x/exp/slog"
+
 	"github.com/gorilla/websocket"
 
 	"github.com/gnolang/gno/tm2/pkg/amino"
```

Comment:
Perhaps we should include a more effective import linter in the `make fmt` command. This way, it will be executed regularly and tested by the CI.

Reply:
Added this in as part of the previous comment:

https://github.com/gnolang/gno/pull/1302#discussion_r1461649017

Added the `goimports` CI:
[4130b15](https://github.com/gnolang/gno/pull/1302/commits/4130b15b679d4ab22e107927d5f3416ccd1e1b04)

Our linter already picks up unsorted `goimports`, but the Makefile + CI makes it easy to enforce and correct

Reply:
See https://github.com/gnolang/gno/pull/1302#pullrequestreview-1839642325.